### PR TITLE
ISPN-14623 ZSCORE

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-14623

### CODE REVIEW

File: server/resp/src/test/java/org/infinispan/server/resp/SortedSetCommandsTest.java [hunk 1]
```diff
@@ -240,4 +240,19 @@ public void testZPOPMAX() {
...
+            ScoredValue.just(-10, "tristan"),
+            ScoredValue.just(1, "ryan"),
+            ScoredValue.just(17, "vittorio"),
+            ScoredValue.just(18.9, "fabio"),
+            ScoredValue.just(18.9, "jose"),
+            ScoredValue.just(18.9, "katia"),
+            ScoredValue.just(21.9, "marc"));
+      assertThat(redis.zscore("people", "jose")).isEqualTo(18.9);
+      assertThat(redis.zscore("people", "takolo")).isNull();
+      assertWrongType(() -> redis.set("another", "tristan"), () ->  redis.zpopmax("another"));
```
Comment (jabolina) [inline server/resp/src/test/java/org/infinispan/server/resp/SortedSetCommandsTest.java:256]:
```suggestion
      assertWrongType(() -> redis.set("another", "tristan"), () ->  redis.zscore("another"));
```

**Review by jabolina — COMMENTED** [review]
Really small thing just for completeness. LGTM.

**Comment by karesti** [comment]
@jabolina done

**Review by jabolina — APPROVED** [review]

**Comment by karesti** [comment]
@jabolina rebase done

**Review by jabolina — APPROVED** [review]

**Comment by jabolina** [comment]
Integrated, thanks @karesti!

**Pull request merged**